recipe/bld.bat

```diff
@@ -1,2 +0,0 @@
-"%PYTHON%" setup.py install --single-version-externally-managed --record record.txt
-if errorlevel 1 exit 1
```

Comment:
Currently, there is no Windows support. There probably should (and can) be. However, I don't think I am the right person to tackle this problem.